### DISCUSSION TITLE
Help with defining aliases for commands used by mason

### DISCUSSION THREAD

**Discussion by anoduck** [mason-org/mason.nvim#1299] [Q&A] [answered]
Working on getting mason installed, and I need the ability to alias a few commands in lua for my init.lua, so mason will have access to the proper command. For Bash it would be:

```bash
alias tar="/usr/local/bin/gtar"
alias java="/usr/local/jdk-17/bin/java"
alias javac="/usr/local/jdk-17/bin/javac"
```

**Comment by williamboman** [comment, marked as answer]
For the time being you'll have to manually manage your PATH and/or symlinks in such a way that these executables have precedence.
